docs/explanation/bases.rst

```diff
@@ -1,68 +1,66 @@
+
+Setting an Ubuntu base for a rock is especially useful when the goal is to build a rock
+that can be used for general-purpose environments, such as a development working space.
+
+
+LTS and interim bases
+---------------------
 
-The resulting rock will therefore also contain any software and utilities that
-are present in the Ubuntu base container images.
+Ubuntu bases are divided into *LTS* and *interim* bases. An LTS base contains an Ubuntu
+LTS release and has a 10-year support window. An interim base contains an interim Ubuntu
+release and is supported for nine months.
+
+Interim bases are less stable than LTS bases because they contain experimental features
+and have shorter lifespans. They are most suitable for container-based testing the
+Ubuntu release itself, using it as a proving ground for upcoming features and
+compatibility.
 
-Setting an Ubuntu :ref:`base <rockcraft_yaml_base>` for a rock is
-especially useful when the goal is to build a rock that can be used for
-general-purpose environments such as a development working space.
 
 Bare bases
 ----------
 
-As the name suggests, a ``bare`` :ref:`base <rockcraft_yaml_base>` indicates
-that the rock shall have no
-baseline system. This definition is similar to the "`scratch`_" Docker image,
-with the exception that a rock **can never be completely empty**, as it must
-always include a minimum baseline with :ref:`pebble_explanation_page`
-and additional metadata (:ref:`what-sets-rocks-apart`).
-
-When and how to use a ``bare`` base
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The ``bare`` :ref:`base <rockcraft_yaml_base>` is extremely useful when
-the goal is to build a lean **chiselled rock**. Such rocks are typically
-preferred for production environments due to their optimised deployment
-efficiency and reduced attack surface. The combination of the
-``bare`` :ref:`base <rockcraft_yaml_base>`
-with :ref:`Chisel <chisel_explanation>` slices will result in a minimalist
-container image that meets the production environment's criteria while
-retaining its functionality.
-
-Note that, when using the ``bare``
-:ref:`base <rockcraft_yaml_base>`, the rock will still be
-**Ubuntu-based**! As such, Rockcraft needs to know the Ubuntu release to be used
-as the :ref:`build environment <rockcraft_yaml_build_base>` for the rock.
+As the name suggests, a *bare* base results in a rock with no baseline system. This
+definition is similar to the scratch Docker image, with the exception that a rock *can
+never be completely empty*, as it must always include a minimum baseline with
+:ref:`pebble_explanation_page` and :ref:`additional metadata <what-sets-rocks-apart>`.
+
+The bare base is extremely useful when the goal is to build a lean chiselled rock.
+Such rocks are typically preferred for production environments due to their optimized
+deployment efficiency and reduced attack surface. The combination of the bare base with
```

Comment:
I know this isn't rewording the previous text, but I'm not sure I agree with this. I've historically switched from very small Alpine bases for Docker to much bigger Ubuntu-based container images specifically because the shared layers resulted in a saving of storage space and memory.

Bare bases are very useful though when the rock is the product (e.g. a rock containing a database server) rather than just a means of deployment (e.g various microservices for a single website).

Reply:
Assuming the exception is with the "optimized deployment efficiency", then I see Alex's point but I don't think this is a change that needs to happen on this PR specifically (it's something we should bring up with Cris, he wrote the original text I believe).